test/services/user/unit.test.js

```diff
@@ -97,6 +98,17 @@ describe('publicTeachers service', () => {
 		params = await testObjects.generateRequestParamsFromUser(teacherFromDifferentSchool);
 	});
 
+	after((done) => {
+		server.close(done);
+	});
+	
```

Comment:
Codacy found an issue: [Trailing spaces not allowed.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5680233)